lib/systems/inspect.nix

```diff
@@ -91,6 +91,10 @@ rec {
     isMusl         = with abis; map (a: { abi = a; }) [ musl musleabi musleabihf muslabin32 muslabi64 ];
     isUClibc       = with abis; map (a: { abi = a; }) [ uclibc uclibceabi uclibceabihf ];
 
+    isEfiEnvironment = [
```

Comment:
How about `isUefi`?

Reply:
It's weirdly close to `isEfi` which makes me meh, but why not.

Reply:
Well all that matters is that isn't exactly the same, so that this PR can merge concurrently with the deprecation period for `isEfi`.

Also in general for every kernel `*-*-kernelname` we have `.isKernelName`, so having the predicate for `*-*-uefi` be `isUefi` is definitely "principle of least surprise".